schema/base_schema.json5

```
@@ -48,67 +43,70 @@
+    "FailureEvent": {
+      "description": "A NonInteractiveEvent that is sent when a user action results in a error, \n\
+        like an invalid email when sending a form.",
+      "parents": [
+        "NonInteractiveEvent"
+      ],
+      "requiresContext": [],
+      "properties": {
+        "message": {
+          "description": "Failure message.",
+          "type": "string"
+        }
+      },
     },
-    "URLChangeEvent": {
-      "description": "\
-        non interactive event that is emitted when the URL of a page has changed. Also contains a `WebDocumentContext` \n\
-        that details the change.",
-      "parents": ["NonInteractiveEvent"],
-      "requiresContext": ["WebDocumentContext"]
+    "InputChangeEvent": {
+      "description": "Event triggered when user input is modified.",
+      "parents": ["InteractiveEvent"],
+      "requiresContext": ["InputContext"]
     },
-    "ApplicationLoadedEvent": {
-      "description": "\
-        non interactive event that is emitted after an application (eg. SPA) has finished loading. \n\
-        Contains a `SectionContext`",
+    "InteractiveEvent": {
+      "description": "The parent of Events that are the direct result of a user interaction, e.g. a button click.",
+      "parents": ["AbstractEvent"],
+      "requiresContext": ["SectionContext"]
+    },
+    "NonInteractiveEvent": {
+      "description": "The parent of Events that are not directly triggered by a user action.",
+      "parents": ["AbstractEvent"],
+      "requiresContext": []
+    },
+    "PressEvent": {
+      "description": "An InteractiveEvent that is sent when a user presses on a pressable element \n\
+        (like a link, button, icon).",
+      "parents": ["InteractiveEvent"],
+      "requiresContext": ["PressableContext."]
+    },
+    "SectionHiddenEvent": {
+      "description": "A NonInteractive  event, emitted after a section (`SectionContext`) has become invisible.",
       "parents": ["NonInteractiveEvent"],
       "requiresContext": ["SectionContext"]
     },
     "SectionVisibleEvent": {
-      "description": "Non interactive event, emitted after a section (`SectionContext`) has become visible.",
+      "description": "A NonInteractive  event, emitted after a section (`SectionContext`) has become visible.",
       "parents": ["NonInteractiveEvent"],
       "requiresContext": ["SectionContext"]
     },
-    "SectionHiddenEvent": {
-      "description": "\
-        Non interactive event, emitted after a section (`SectionContext`) has become invisible.",
+    "SuccessEvent": {
+      "description": "A NonInteractiveEvent that is sent when a user action is successfully completed, \n\
+        like sending an email form.",
       "parents": ["NonInteractiveEvent"],
-      "requiresContext": ["SectionContext"]
+      "requiresContext": [],
+      "properties": {
+        "message": {
+          "description": "Success message.",
+          "type": "string"
+        }
     },
```

There's a missing curly bracket here, closing the properties object:

```suggestion
      }
    },
```